api.lua

```diff
@@ -127,7 +127,7 @@ local function fire(stack, player)
 			dmg = dmg * 1.2
 		end
 
-		target:set_hp(target:get_hp() - dmg)
+		target:punch(player, nil, {damage_groups={fleshy=dmg}})
```

Comment:
Please add spaces around `=`s
```suggestion
		target:punch(player, nil, {damage_groups = {fleshy = dmg}})
```